.gitignore

```diff
@@ -7,6 +7,7 @@
 bin/**
 **/__debug_bin
 **/__pycache__
+**/venv
```

Comment:
I don't hate this, but it assumes everyone uses the same name for their python venv as you.

You can set up personal gitignores in several ways. For example, my ~/.gitconfig contains:

```
[core]
	excludesfile = /home/efried/.gitignore
```

...and that file contains:

```
.vscode/
*.orig
.sw[a-p]
.*.sw[a-p]
```

(The idea being that not everyone necessarily uses vscode -- though I see we've got that in here on L33 -- or vim.)